### PR TITLE
Correct input collections to the TF emulators for Run 3 MC re-emulation

### PR DESCRIPTION
**Pull request by eyigitba**
#### PR description:

This PR is to fix the input parameters for TFs for Run 3 MC re-emulation. The inputs are now being provided by the TP emulators which are called during the MC re-emulation workflow.

This change reverts https://github.com/cms-l1t-offline/cmssw/issues/886 for Run 3 MC only and fixes the CPPFEnable flag for EMTF which should remain False for MC.

#### PR validation:
Checked with:
SingleNeutrino_E-10-gun/Run3Winter22DR-L1TPU0to99FEVT_SNB_122X_mcRun3_2021_realistic_v9-v2/GEN-SIM-DIGI-RAW

The re-emulation works and produces the correct collections.

### CODE REVIEW

**Comment by epalencia** [comment]
Please test

**Comment by eyigitba** [comment]
I don't think this failed test is related to the change. Should we launch the tests again?

File: L1Trigger/Configuration/python/customiseReEmul.py [hunk 1]
```diff
@@ -284,13 +284,31 @@ def L1TReEmulMCFromRAW(process):
...
         srcDTPh  = 'simDtTriggerPrimitiveDigis',
         srcDTTh  = 'simDtTriggerPrimitiveDigis'
     )
 
+    run3_GEM.toModify(process.simEmtfDigis,
+      RPCInput  = cms.InputTag('muonRPCDigis'),
+      GEMEnable = cms.bool(False),  # Will be enabled when GEM is in use.
+      GEMInput  = cms.InputTag('simMuonGEMPadDigiClusters'),
+      CPPFEnable = cms.bool(False), # Use CPPF-emulated clustered RPC hits from CPPF as the RPC hits. Set to "False" for MC
+      UseRun3CCLUT_OTMB = cms.bool(False), # TODO: Enable UseRun3CCLUT_OTMB once it's ready.
```
Comment (perrotta) [inline L1Trigger/Configuration/python/customiseReEmul.py:309]:
```suggestion
      RPCInput  = 'muonRPCDigis',
      GEMEnable = False,  # Will be enabled when GEM is in use.
      GEMInput  = 'simMuonGEMPadDigiClusters',
      CPPFEnable = False,  # Use CPPF-emulated clustered RPC hits from CPPF as the RPC hits. Set to "False" for MC
      UseRun3CCLUT_OTMB = False,  # TODO: Enable UseRun3CCLUT_OTMB once it's ready.
```

**Comment by perrotta** [comment]
please test

**Comment by cecilecaillol** [comment]
+l1

**Comment by qliphy** [comment]
+1

**Pull request merged**